Formula/pcre.rb

```diff
@@ -30,6 +30,11 @@ class Pcre < Formula
   uses_from_macos "bzip2"
   uses_from_macos "zlib"
 
+  def want_jit?
```

Comment:
This should just be a variable at best. Not a method.

Reply:
Eliminated as a function.